documentation/ame-guide/modules/ROOT/pages/edit-models.adoc

```diff
@@ -128,6 +131,82 @@ For a new model that only has an Aspect so far:
 
 image::different-constraints.png[Different Constraints]
 
+[[abstract-entity-case]]
+=== Use Abstract Entities
+
+There are two different ways to create an abstract entity:
+
+* Use drag-and-drop from the Workspace sidebar
+* By adjusting an Entity in the extends field
+
+*Use drag-and-drop from the Workspace sidebar To add abstract entity, proceed as follows:*
```

Comment:
*Use drag-and-drop from the Workspace sidebar To add an abstract entity, proceed as follows:*